$NetBSD: patch-ab,v 1.2 2022/12/08 08:51:56 vins Exp $

Depend on [n]curses to provide prototypes for tparm and tgoto.

--- we_debug.c.orig	2005-07-07 01:53:09.000000000 +0000
+++ we_debug.c
@@ -62,10 +62,6 @@ extern BUFFER *e_p_w_buffer;
 extern char *att_no;
 extern char *e_tmp_dir;
 
-#ifdef NOTPARM
-char *tparm();
-char *tgoto();
-#endif
 #ifdef DEFTPUTS
 int tputs();
 #endif
